wagtail/wagtailsnippets/tests.py

```diff
@@ -131,6 +131,19 @@ def test_simple(self):
         response = self.get()
         self.assertEqual(response.status_code, 200)
         self.assertTemplateUsed(response, 'wagtailsnippets/snippets/create.html')
+        self.assertNotContains(response, '<ul class="tab-nav merged">')
```

Comment:
@timheap, `assertNotContains` and `assertContains` with `html=True` checks nodes, so even if `'<ul class="tab-nav merged">something</ul>'` in `response`
```
self.assertNotContains(response, '<ul class="tab-nav merged">', html=True)
```
Will not raise `AssertionError`.

Should I write asserts with `BeautifulSoup`? Or can you recommend something else?